packages/server/billing/helpers/adjustUserCount.ts

```diff
@@ -82,7 +82,8 @@ const addUser = async (orgIds: string[], userId: string) => {
       (oldOrganizationUser && oldOrganizationUser.newUserUntil) ||
       organization.periodEnd ||
       new Date()
-    return new OrganizationUser({orgId, userId, newUserUntil})
+    const tier = organization.tier
+    return new OrganizationUser({orgId, userId, newUserUntil, tier})
```

Comment:
Stylistically I prefer...
`return new OrganizationUser({orgId, userId, newUserUntil, tier: organization.tier}`
...but that might be because I am old and like to avoid defining new vars